src/Kunstmaan/MediaBundle/Controller/MediaController.php

```diff
@@ -472,4 +493,49 @@ public function moveMedia(Request $request)
 
         return new JsonResponse();
     }
+
+    /**
+     * @Route("/bulk-move", name="KunstmaanMediaBundle_media_bulk_move")
+     *
+     * @param Request $request
+     *
+     * @return JsonResponse|Response
+     * @throws \Doctrine\ORM\EntityNotFoundException
+     */
+    public function bulkMoveAction(Request $request)
+    {
+        $em = $this->getDoctrine()->getManager();
+        $mediaRepo = $em->getRepository('KunstmaanMediaBundle:Media');
+        $form = $this->createForm(BulkMoveMediaType::class);
+
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $folder = $form->getData()['folder'];
+            $mediaIds = explode(',', $form->getData()['media']);
+
+            foreach ($mediaIds as $mediaId) {
+                /** @var Media $media */
+                $media = $mediaRepo->getMedia($mediaId);
```

Comment:
Fetching from database in a loop? Not a good idea. A simple `UPDATE` query would do fine here.

Reply:
@mlebkowski Correct! Changed ;)